libres/lib/analysis/enkf_linalg.cpp

```diff
@@ -93,60 +93,55 @@ void enkf_linalg_genX2(matrix_type *X2, const matrix_type *S,
 /*This function is similar to enkf_linalg_svdS but it returns the eigen values without its inverse and also give the matrices truncated U VT and Sig0*/
 
 // Trunc.SVD(S)  = U0 * Sig0 * V0T
-int enkf_linalg_svd_truncation(const matrix_type *S, double truncation,
-                               int ncomp, dgesvd_vector_enum store_V0T,
-                               double *sig0, matrix_type *U0,
-                               matrix_type *V0T) {
+int enkf_linalg_svd_truncation(const matrix_type *S,
```

Comment:
We removed this function in the following commit:

https://github.com/equinor/ert/commit/ba43cfb51327e4a227b580c0cfb9753e7ad63662

Reply:
I'll rebase